linAssgn1

```diff
@@ -0,0 +1,16 @@
+#!/bin/bash
+clear
+echo -n "Enter the Process Id to be killed : " 
+read prid
+if [ $prid -le 1 -o $prid -ge 32768 ]
```

Comment:
I have changed the code, instead of finding the maximum pid by manually running the script the shell script itself will check for the maximum process id using this command 'cat /proc/sys/kernel/pid_max'